data/mods/ssb/abilities.js

```diff
@@ -319,6 +319,50 @@ let BattleAbilities = {
 			}
 		},
 	},
+	// pre
```

Comment:
For alphabetical order, this should be after pablo and before ptoad.

Same thing for status/move.

Reply:
Doh! I screwed this up because of the casing - for example, the statuses file has them as ids so `ptoad` comes before `psynergy` (because its actually supposed to be `Psynergy`. My bad!